src/js/config/dev.js

```diff
@@ -1,4 +1,4 @@
 export default {
-  // host: process.env.EVENTPLOG_API || 'http://techmeetupsng-backend.herokuapp.com'
-  host: process.env.EVENTPLOG_API || 'http://localhost:8000'
+   host: process.env.EVENTPLOG_API || 'http://techmeetupsng-backend.herokuapp.com'
+  //host: process.env.EVENTPLOG_API || 'http://localhost:8000'
```

Comment:
Does this change from time to time? When is the `host` used? Can we remove the commented out code? How do we make this work in a way that the host is what we want it to be based on what we need to achieve?